File: backend/text.py
# Download the helper library from https://www.twilio.com/docs/python/install
import os
from twilio.rest import Client


# Find your Account SID and Auth Token at twilio.com/console
# and set the environment variables. See http://twil.io/secure
account_sid = os.environ['TWILIO_ACCOUNT_SID']
auth_token = os.environ['TWILIO_AUTH_TOKEN']
client = Client(account_sid, auth_token)

message = client.messages \
    .create(
         body='McAvoy or Stewart? These timelines can get so confusing.',
         from_='+15017122661',
         status_callback='http://postb.in/1234abcd',
         to='+15558675310'
     )

print(message.sid)
print(account_sid)

File: backend/text.js
if (process.env.NODE_ENV !== 'production') {
    require('dotenv').config();
  }
const accountSid = process.env.TWILIO_ACCOUNT_SID;
const authToken = process.env.TWILIO_AUTH_TOKEN;
const client = require('twilio')(accountSid, authToken);


client.messages
      .create({
         body: 'Test message.',
         from: process.env.TWILIO_NUMBER,
         to: process.env.MY_PHONE_NUMBER,
         statusCallback: 'https://script.google.com/macros/s/AKfycbzAl66Dpm4eHdkG4iDmR-0Sdu_3RcXQ-ehnJLRvE1aPUiMVhtb3eVgZE-vsVcvjwhFM/exec'
       })
      .then(message => console.log(message))
      .catch(err => console.log(err));